tests/integration/targets/route53_health_check/tasks/calculate_health_check.yml

```diff
@@ -0,0 +1,141 @@
+        type: CALCULATED
+        health_threshold: 1
+        child_health_checks: '{{ create_result.health_check.id }}'
+      register: create_calculated
+    
+    - name: 'Check result - Calculated Health Check'
+      assert:
+        that:
+        - create_calculated is not failed
+        - create_calculated is changed
+
+    - name: 'Check result - Create Calculated Health Check - idempotency'
+      route53_health_check:
+        health_check_name: "calculated_health_check"
+        use_unique_names: true
+        type: CALCULATED
+        health_threshold: 1
+        child_health_checks: '{{ create_result.health_check.id }}'
+      register: create_idem
+    
+    - name: 'Check result - Calculated Health Check - idempotency'
+      assert:
+        that:
+        - create_idem is not failed
+        - create_idem is not changed
+
+    - name: 'Update Calculated Health Check'
+      route53_health_check:
+        health_check_name: "calculated_health_check"
+        use_unique_names: true
+        type: CALCULATED
+        health_threshold: 2
+        child_health_checks: '{{ create_result.health_check.id }}'
+      register: check_updated_calculated
+      check_mode: true
+    
+    - name: 'Check result - Update Calculated Health Check - check_mode'
+      assert:
+        that:
+        - check_updated_calculated is not failed
+        - check_updated_calculated is changed
+    
+    - name: 'Update Calculated Health Check'
+      route53_health_check:
+        health_check_name: "calculated_health_check"
+        use_unique_names: true
+        type: CALCULATED
+        health_threshold: 2
+        child_health_checks: '{{ create_result.health_check.id }}'
+      register: updated_calculated
+
+    - name: 'Check result - Update Calculated Health Check'
+      assert:
+        that:
+        - updated_calculated is not failed
+        - updated_calculated is changed
+    
+  # Cleanup starts here =================================================================
+  always:
+      - name: 'Delete Calculated Health Check'
+        amazon.aws.route53_health_check:
+          state: absent
+          health_check_id: '{{ create_calculated.health_check.id }}'
+        register: deleted_calculated
+        
+      - assert:
+          that:
+          - deleted_calculated is not failed
+          - deleted_calculated is changed
+        
+      - name: 'Delete HTTP health check with use_unique_names'
+        amazon.aws.route53_health_check:
+          state: absent
+          name: '{{ tiny_prefix }}-{{ resource_path }}-test-hc-tag-operations'
+          ip_address: '{{ ip_address }}'
+          port: '{{ port }}'
+          type: '{{ type_http }}'
+          resource_path: '{{ resource_path }}'
+          use_unique_names: true
+          fqdn: '{{ fqdn }}'
+        register: delete_result
+
+      - assert:
+          that:
+            - delete_result is changed
+            - delete_result is not failed
```

Comment:
These tasks should be moved to the block above, then add two tasks to the always block that delete the two health checks and use `ignore_errors: true`. Even though the tasks you have here delete things, they are testing delete functionality and should be separate from cleanup tasks.